### DISCUSSION TITLE
Where's the linter report?

### DISCUSSION THREAD

**Discussion by iapazmino** [super-linter/super-linter#5575] [Q&A] [answered]
Hello,

I'm trying to get the report resulting from executing super-linter github action.
So far all I've been able to acquire is the log file, but that file includes everything that happened in the build so it's not really useful for my usecase.

Where is the linter report?

Thanks,
Iván

**Comment by ferrarimarco** [comment]
Hi @iapazmino !

What do you mean by "linter report" exactly?

**Reply by iapazmino** [reply]
when you run a test coverage tool you will print to the log stuff like
finding production code
finding test code
computing coverage 
classOne.java 70%
classTwo.java 77%
... and so on

but you will also find a file, coverage.xml for instance, with a report of computing coverage; just the results, not the steps in the process of computing the coverage.

When the super-linter git hub actions runs the action log prints a lot of things about super-linter initialization and then this line "Starting audit ..." that signals the beginning of the results and another one that signals the end "Audit done."

Where is the file that contains only the results of the audit or what is the configuration I need to adjust to produce it?

**Reply by ferrarimarco** [reply, marked as answer]
There's currently no support for this in super-linter.

**Reply by iapazmino** [reply]
Got it, thanks.
